src/core/VLIW/DependencyChecker.ts

```diff
@@ -52,7 +52,10 @@ export class DependencyChecker {
             case Opcodes.SF:
             case Opcodes.BEQ:
             case Opcodes.BNE:
+            case Opcodes.BGT:
+                break;
             default:
+                throw new Error("Error at checkTargetOperation, unknown opcode: " + Opcodes[operation.opcode]);
```

Comment:
Remember that throwing exceptions cause the following code to be unreachable! I'll point out all cases where this happens in your PR too

Reply:
Ooops... My bad. IDK what I was thinking. Fixing in the next commit.